app/models/parcel.rb

```diff
@@ -3,14 +3,20 @@ class Parcel < ActiveRecord::Base
   belongs_to :sender_info
   belongs_to :recipient_info
 
+  enum acceptance: {
```

Comment:
Może jakąś lepszą nazwę na to? `status`? A może `acceptance_status`? Wydaje mi się ciut bardziej sensowne i lepiej odzwierciedlające intencje.
(Zwłaszcza że jest walidator `acceptance`)